includes/database/tables/class-order-adjustments.php

```diff
@@ -64,7 +65,8 @@ protected function set_schema() {
 		$this->schema = "id bigint(20) unsigned NOT NULL auto_increment,
 		object_id bigint(20) unsigned NOT NULL default '0',
 		object_type varchar(20) DEFAULT NULL,
-		type_id bigint(20) unsigned NOT NULL default '0',
+		type_id bigint(20) unsigned DEFAULT NULL,
+		type_key varchar(255) DEFAULT NULL,
```

Comment:
This is really unimportant, but can you move `type_key` to after `type` ? This will need to be done:

1. Here
2. In the migration further down.
3. In the schema file.

I just like having `type_id` and `type` next to each other. :P